docs/src/components/GuidesNavigationFooter.tsx

```diff
@@ -19,7 +19,7 @@ const GuidesNavigationFooter: React.FC<GuidesNavigationFooterProps> = ({ previou
     <br />
     <Flex gap="gap.small">
       {previous && (
-        <Button
+        <Button<Link>
```

Comment:
This is unacceptable solution...

Reply:
why? This is an acceptable from TS point of view - please, be specific about your criterias when using words like 'unacceptable'

Reply:
- `<Button as={Link} />` - fine
- `<Button<Link> />` - fine
- `<Button<Link> as={Link} />` - too noisy

I want to find a way where we can go with the first option